### DISCUSSION TITLE
Copilot interfering with vscode snippets

### DISCUSSION THREAD

**Discussion by moritztim** [community/community#39464] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When trying to fill out a snippet, I want to press `tab` to get to the next cursor position, but instead copilot jumps in and completes the entire block.
I don't know the best way of solving this but one approach might be to use a different shortcut for accepting completions when there's a conflict with another shortcut, like jumping to the next cursor in a snippet.

I was not able to properly search for simmilar issues, the search `snippets category:Copilot` returned the same as `category:Copilot`.

**Comment by iamcjwilliams** [comment]
@moritz-t-w Thank you for your feedback, this is on our copilot team's radar! 🚀

**Comment by asifm** [comment]
> but one approach might be to use a different shortcut

Indeed. I changed Copilot's trigger to `alt + a` (following the suggestion [here](https://stackoverflow.com/a/73687491/1927583)). This shortcut makes sense because the other Copilot shortcuts also start with `alt`. It may take a little time to get used to, though.
